### DISCUSSION TITLE
Only page 1 works in the recently updated ForgeUI

### DISCUSSION THREAD

**Discussion by grandfromages** [BlafKing/sd-civitai-browser-plus#327] [Q&A]
8/20/2024 - This still works in the newly-updated ForgeUI, but only for page 1. 
"Next page" is broken, and the slider to pick >1 is limited to only 1. 
This is a wonderful mod, so please keep up the good work! :)

**Comment by FifthDream** [comment]
Same issue. I have found that if i click on the "next page" button, numerous times in different locations on the button (spamming clicks), it will eventually load the next page. That's not really a solution, but hopefully it will help finding a solution, and is a bit of a workaround until it's remedied.
